src/net/sourceforge/kolmafia/persistence/ConsumablesDatabase.java

```diff
@@ -133,154 +124,94 @@ public record DustyBottle(int id, String name, String alias) {}
         ItemPool.DUSTY_BOTTLE_OF_MUSCAT, "dusty bottle of Muscat", "dusty bottle of bad Muscat")
   };
 
-  private static final Map<String, String> muscleByName = new HashMap<String, String>();
-  private static final Map<String, String> mysticalityByName = new HashMap<String, String>();
-  private static final Map<String, String> moxieByName = new HashMap<String, String>();
-
   private ConsumablesDatabase() {}
 
   public static void reset() {
     ConsumablesDatabase.readConsumptionData(
-        "fullness.txt", KoLConstants.FULLNESS_VERSION, ConsumablesDatabase.fullnessByName);
+        "fullness.txt", KoLConstants.FULLNESS_VERSION, KoLConstants.CONSUME_EAT);
     ConsumablesDatabase.readConsumptionData(
-        "inebriety.txt", KoLConstants.INEBRIETY_VERSION, ConsumablesDatabase.inebrietyByName);
+        "inebriety.txt", KoLConstants.INEBRIETY_VERSION, KoLConstants.CONSUME_DRINK);
     ConsumablesDatabase.readConsumptionData(
-        "spleenhit.txt", KoLConstants.SPLEENHIT_VERSION, ConsumablesDatabase.spleenHitByName);
+        "spleenhit.txt", KoLConstants.SPLEENHIT_VERSION, KoLConstants.CONSUME_SPLEEN);
     ConsumablesDatabase.readNonfillingData();
   }
 
   static {
+    for (int i = 0; i < 1 << 5; ++i) {
```

Comment:
1 << 5 is basicly a magic number here - split into a named const or calculate it staticly too

Reply:
Just noticed it was always there. Still.

Reply:
Done.

Reply:
🎉